sorting/Bogo_Sort.cpp

```diff
@@ -22,7 +32,8 @@ bool is_sorted(const std::vector <int> &arr)
     return true;
 }
 //This function prints the array
-void show_array(const std::vector <int> &arr)
+template<typename T, size_t N>
```

Comment:
function documentation
```suggestion

/**
 * Function to display array on screen 
 * @tparam T typename of the array
 * @tparam N length of array
 * @param arr array to display
 */
template<typename T, size_t N>
```